local/sites/test-site/env.yml

```diff
@@ -18,25 +18,22 @@ agent:
 metadata:
   name: detroit-preprod
   kind: pre-prod
-
+  etcd_initial_cluster: etcd-c1=http://10.10.10.10:2380,etcd-c2=http://10.10.10.11:2380,etcd-c3=http://10.10.10.12:2380
```

Comment:
not the exact cluster, off by 1 for the core-cloud demo. should go from `.11-.13`